doc/source/WorkingWithScenes/Camera/screengrab.py

```diff
@@ -0,0 +1,263 @@
+# BuildTarget: images/interfaceCameraVisualizer.png
+
+import os
+import subprocess32 as subprocess
+import tempfile
+import time
+
+import imath
+import IECore
+import Gaffer
+import GafferScene
+import GafferUI
+import GafferSceneUI
+
+scriptWindow = GafferUI.ScriptWindow.acquire( script )
+viewer = scriptWindow.getLayout().editors( GafferUI.Viewer )[0]
+graphEditor = scriptWindow.getLayout().editors( GafferUI.GraphEditor )[0]
+hierarchyView = scriptWindow.getLayout().editors( GafferSceneUI.HierarchyView )[0]
+
+# Delay for x seconds
+def __delay( delay ) :
+	endtime = time.time() + delay
+	while time.time() < endtime :
+		GafferUI.EventLoop.waitForIdle( 1 )
+
+# Create a random directory in `/tmp` for the dispatcher's `jobsDirectory`, so we don't clutter the user's `~gaffer` directory
```

Comment:
Someone with Python know-how should probably take a glance between here and L44.